hw/top_earlgrey/ip/ast/ast_wrapper.core

```diff
@@ -9,6 +9,8 @@ filesets:
     depend:
       - lowrisc:systems:ast_wrapper_pkg
       - lowrisc:systems:ast
+      - lowrisc:top_earlgrey:clkmgr
```

Comment:
```suggestion
      - lowrisc:top_earlgrey:clkmgr_pkg
```

should be sufficient.

Reply:
Correct, thanks. I changed this as well.